app/views/transfer/_thesis_target.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr>
```

Comment:
I think we'll want the author kerb displayed here. The file names are _supposed_ to include a kerb and that will be a really easy way for a processor to rapidly identify which files are most likely what they are looking for. If we need to remove something to fit it, I suspect at this stage the kerb is more important than degree (I would think that is more of a thesis processing step), but Jess could confirm whether my suspicious is correct.

Reply:
I've pushed an update to this, adding the user's kerberos to this line - so it is new `kerbid: display_name`